sponsorspartners.html

```diff
@@ -258,6 +258,16 @@ <h2 class="occupation" style=color:#575150>Workshop Sponsor</h2>
                 <img src = "images/Sponsor Logos/Adobe_Corporate_Horizontal_Lockup_Red_HEX.jpg" style = "padding: 50px; max-width: 300px;">
                 </a>
             </figure>
+            <figure class = "u-block-hover">
+                <a href = "https://research.facebook.com/">
+                <img src = "images/Sponsor Logos/Meta_lockup_positive_primary_RGB.png" style = "padding: 50px; max-width: 300px;">
```

Comment:
We may need to adjust padding-top by pixels. Not they are not aligned in same horizontal level.

Reply:
I've changed the padding and EY logo, they are pretty aligned on my devices